package.json

```
@@ -5,9 +5,11 @@
   "type": "module",
   "license": "AGPL-3.0",
   "engines": {
-    "node": ">=20"
+    "node": ">=18"
   },
+  "packageManager": "pnpm@8.6.6",
```

Biome (2.1.2)

[error] The key packageManager was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)